.lintr

```diff
@@ -1,2 +1,2 @@
-linters: linters_with_defaults(line_length_linter(150), object_usage_linter=NULL, cyclocomp_linter(complexity_limit = 20))
-exclusions: list("R/data.R")
+linters: linters_with_defaults(line_length_linter(150), object_usage_linter=NULL, cyclocomp_linter(complexity_limit = 20), commented_code_linter = NULL)
```

Comment:
https://lintr.r-lib.org/reference/exclude.html#details

Reply:
@bms63, quite a few code blocks in Pilot3 with commented codes and 'unexpected chars' to apply nolints to address this item in a '_by section_' manner where we don't want lintr to check. I'll chip away at it using the same branch and PR. thx.

Reply:
ummm.. i wonder if there is a way to globally set to check after line 20 or something??

Reply:
i don't see anything - lets just leave as it.  I didn't realize we would need so many of them

Reply:
Yes, there is @bms63 Please see [here](https://github.com/r-lib/lintr/blob/HEAD/R/exclude.R). 

What do you say, we should apply lintr > line 20. Let's give it a try and re-enable the commented_code_linter. thx.

Reply:
oh way cool!!  yeah I think we shouldn't have commented out code in our scripts, but since we have program headers we should tell the lintr not to check them . maybe set it to whatever the biggest header is?